wp-content/themes/humanity-theme/theme.json

```diff
@@ -3,6 +3,50 @@
   "version": 3,
   "settings": {
     "appearanceTools": false,
+    "blocks": {
+    "core/quote": {
+      "color": {
+        "text": true,
+        "palette": [
+          {
+            "slug": "black",
+            "color": "#000",
+            "name": "Black"
+          },
+          {
+            "slug": "white",
+            "color": "#fff",
+            "name": "White"
+          },
+          {
+            "slug": "amnesty-grey_brand",
+            "color": "#b3b3b3",
+            "name": "Grey Brand"
+          }
+        ]
+      },
+      "typography": {
+        "fontSizes": [
+          {
+            "slug": "small",
+            "size": "1.75rem",
+            "name": "Small"
+          },
+          {
+            "slug": "medium",
+            "size": "2rem",
+            "name": "Medium"
+          },
+          {
+            "slug": "large",
+            "size": "2.25rem",
+            "name": "Large"
+          }
+        ],
```

Comment:
I see that the [existing blockquote styles](https://github.com/amnestywebsite/humanity-theme/blob/develop/private/src/styles/blocks/core-blocks/_blockquote.scss#L104) for font sizes are responsive. Perhaps adding [fluid typography settings](https://developer.wordpress.org/themes/global-settings-and-styles/settings/typography/#enabling-fluid-typography) here would transition the styles better, allowing a range of sizes for each preset.

Also, do some of those existing styles need to be removed? I notice some competing with the core styles (for example, set the core quote block text size to XL, and see that the paragraph font-size goes to 36px, rather than the built in x-large preset of 42px).

Reply:
> Also, do some of those existing styles need to be removed?

Realizing now that's not an option, since there will still be instances of the old block.